### ISSUE TITLE
Hostile Apparatus: Comms Center - 2 major issues due to poi resetting while inside. 

### ISSUE THREAD

**Issue by PoorLuckPierre** [RavienGaming/Reforged-Eden-2#172]
### Reforged Eden 2 Build Number

14

### Game mode

Dedicated Server

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

I was playtesting the storyline on a mp server for the first time. During Hostile Apparatus I encountered some major issues with the POI. 

As someone else had already entered the comm station before me, I went in to complete the quest steps and get the story line finished up. Where needed I skipped a few steps that involved killing or removing spawn plates etc.

After finishing the quest line I was on the last step that tells me to go back to the elevator. 

On my way back out the POI reset. This meant that when I tried to leave through the incinnerator security room the doors would close from the entry side and the thrusters would turn on to boil me. The security (mainframe) side door stayed open however. Once I spawned again inside the POI I was faced with shielded turrets. 

When I decided to home spawn and come back to the POI I was able to get back to the incinerator  security room. However once I got back to the room, the mainframe side door was now closed. Without having access to the quest I was not able to get back to the mainframe side.

To prevent this 2 things can be done:

Add a way to get past the security room from the entry side without needing the quest active. I believe RE 1 had a debug panel near the door for this reason. 

Add a teleporter to the last room of the POI that can teleport you back to the comm tower outside of the POI. 

TLDR:

The security room should not be able to lock people in or out of the POI regardless of quest status or POI regeneration. Being locked on the inside of the POI has also happened to me once in RE 1 on a server so it's not a problem unique to RE 2.

**Comment by VermillionX** [comment]
POIs aren't meant to regenerate while a player is in or near them.
This would be a game bug, since POI regeneration is a non-scenario function.
The POI will not be changed to accomodate a one-off game bug that occurs only in MP under extremely specific circumstances that would require the signal logic be redone for the entire POI.

**Comment by PoorLuckPierre** [comment]
Just adding a tp pad in the final room to get back outside would get most of the job done though, if nothing else to prevent people from getting stuck on the inside. I am not sure if that would also generate a lot of work for you guys though.

**Comment by RavienGaming** [comment]
Unfortunately that's not the only place and not the only POI you can get trapped inside if it regenerates while you are in it. Literally any shielded POI will be a death sentence if it regenerates while you are in it as your hand weapons won't be enough to take down a POI shield.
It's not feasible to add an escape route to every room in every POI, and adding it to one off situations like this isn't going to be worth the time as that only fixes that specific room in that specific POI and we can't fix it every time this issue comes up.
I will attach the directions for reporting a bug to the game developers so they can fix POI regenerating when a player is inside of it.

This is likely a game bug and will need to be reproduced and reported using a vanilla save game.
When reproducing a game bug in a vanilla save, make sure you are using the same mode _(singleplayer/co-op/dedicated multiplayer, survival/creative)_ as what you were using when the issue happened in the scenario.
To see a list of known game bugs or to report a new one please see: https://empyriononline.com/forums/bugs.24/
